src/Lykke.Service.PayAuth/Controllers/EmployeesController.cs

```diff
@@ -229,12 +225,11 @@ public class EmployeesController : Controller
             {
                 await _employeeCredentialsService.UpdatePinHashAsync(model.Email, model.PinHash);
             }
-            catch (InvalidOperationException ex)
+            catch (InvalidOperationException e)
             {
-                await _log.WriteWarningAsync(nameof(EmployeesController), nameof(UpdatePinHash), model.ToJson(),
-                    ex.Message);
+                _log.Warning(e.Message, e, model);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.